src/test/java/org/candlepin/resource/util/ConsumerBindUtilTest.java

```diff
@@ -395,13 +395,10 @@ public void handleActivationKeysWithoutAutoAttachUsingSCAWhenBindFails() throws
 
         ConsumerBindUtil consumerBindUtil = this.buildConsumerBindUtil();
 
-        doThrow(new ForbiddenException("exception")).when(this.entitler)
-            .bindByPoolQuantity(eq(consumer), eq(pool1.getId()), eq(5));
-
```

Comment:
since now we'll never get into the `bindByPoolQuantity` method in SCA mode, this exception would never get thrown, so this test is no longer needed, so we can remove it

Reply:
done